src/test/java/bio/terra/common/TestUtils.java

```diff
@@ -120,12 +124,30 @@ public static String validateDrsAccessMethods(List<DRSAccessMethod> accessMethod
   }
 
   public static void verifyHttpAccess(String url, Map<String, String> headers) {
+    verifyHttpAccess(url, headers, true);
+  }
+
+  /**
+   * Make sure that the HTTP url is valid.
+   *
+   * @param shouldAssertAccessibility if true, also assert that the HTTP url is accessible. When
+   *     debugging certain test failures, disabling the accessibility check can allow the remainder
+   *     of the test to complete.
+   */
+  public static void verifyHttpAccess(
+      String url, Map<String, String> headers, boolean shouldAssertAccessibility) {
     HttpUriRequest request = new HttpHead(url);
     headers.forEach(request::setHeader);
     try (CloseableHttpClient client = HttpClients.createDefault()) {
-      try (CloseableHttpResponse response = client.execute(request); ) {
-        assertThat(
-            "Https Uri is accessible", response.getStatusLine().getStatusCode(), equalTo(200));
+      try (CloseableHttpResponse response = client.execute(request)) {
+        int statusCode = response.getStatusLine().getStatusCode();
+        if (shouldAssertAccessibility) {
```

Comment:
If no assertion is done, what is the point of this method? Do we plan to look at the test logs at some point to see if this problem is still occurring?

That is, how is this better than removing the call to `TestUtils.validateDrsAccessMethods()` below?